art/attacks/zoo.py

```diff
@@ -192,7 +192,7 @@ def generate(self, x, y=None):
 
         # Log success rate of the ZOO attack
         logger.info('Success rate of ZOO attack: %.2f%%',
-                    100 * compute_success(self.classifier, x, y, x_adv, self.targeted))
+                    100 * (self.classifier, x, y, x_adv, self.targeted))
```

Comment:
Could you please check if this change is intended?

Reply:
Fixed it!